src/test/java/com/hoteldetails/pad/mappers/hoteldescription/request/DescriptionRequestMapperTest.java

```diff
@@ -0,0 +1,47 @@
+package com.hoteldetails.pad.mappers.hoteldescription.request;
+import com.hotel.service.common.Context;
+import com.hotel.service.description.HotelDescriptionRequest;
+import com.hoteldetails.pad.mappers.common.request.PosMapper;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.opentravel.ota._2003._05.OTAHotelDescriptiveInfoRQ;
+import static org.assertj.core.api.Assertions.assertThat;
+
+
+
+@ExtendWith(MockitoExtension.class)
+class DescriptionRequestMapperTest {
+
+    @Mock
+    PosMapper posMapper;
+    @Mock
+    DescriptiveInfoMapper descriptiveInfoMapper;
+
+    @InjectMocks
+    DescriptionRequestMapper descriptionRequestMapper;
+
+    @Test
+    public void map() {
+        HotelDescriptionRequest hotelDescriptionRequest = getDescriptionRequest();
+        OTAHotelDescriptiveInfoRQ response = descriptionRequestMapper.map(hotelDescriptionRequest);
+        assertThat(response).isNotNull();
```

Comment:
You need to assert all the values that setting inside your subject class

Reply:
also you have to given when conditions for posMapper and descriptiveInfoMapper and verify the interactions